### PR TITLE
Test docker deploy

### PR DESCRIPTION
**Pull request by Romanticoseu**
### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

File: .github/workflows/Shire-Deploy-Docker.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: Docker deploy Shire
```
Comment (glorysdj) [inline .github/workflows/Shire-Deploy-Docker.yaml:1]: Name change to Nightly Build Maven Publish Docker

File: .github/workflows/Shire-Deploy-Docker.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+on:
+  #release:
+  #  types: [created]
+  # schedule:
```
Comment (glorysdj) [inline .github/workflows/Shire-Deploy-Docker.yaml:6]: need cron 0 17 * * *
remove pull request after debugging

File: .github/workflows/Shire-Deploy-Docker.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  #   - cron: '0 15 * * *'
+  # Allows you to run this workflow manually from the Actions tab
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+  
+jobs:
+  deploy-docker:
+
+    runs-on: [self-hosted, Linux, alpha, Shire]
```
Comment (glorysdj) [inline .github/workflows/Shire-Deploy-Docker.yaml:16]: [self-hosted, Shire] is ok

**Comment by Romanticoseu** [comment]
reopen

**Pull request closed without merging**